### PR TITLE
Re-design Featured Sample System

### PR DESCRIPTION
**Pull request by duffh**

### CODE REVIEW

File: src/ArcGISRuntime.Samples.Shared/Managers/SampleManager.cs [hunk 1]
```diff
@@ -67,10 +70,39 @@ public void Initialize()
...
+            using (Stream stream = this.GetType().Assembly.
+                       GetManifestResourceStream("ArcGISRuntime.Resources.FeaturedSamples.xml"))
+            {
+                featuredSampleElement = XElement.Load(stream);
+            }
+
+            // If the resource file has been successfully loaded populate the list of featured samples. 
+            if (featuredSampleElement != null)
+            {
+                featuredSamples = featuredSampleElement.Descendants("Sample").Select(x => x.Value).ToList();
```
Comment (ZackAllen) [inline src/ArcGISRuntime.Samples.Shared/Managers/SampleManager.cs:100]: Alphabetize this list.

**Review by ZackAllen — APPROVED** [review]

File: src/ArcGISRuntime.Samples.Shared/Managers/SampleManager.cs (hunk 1 above)
Comment (duffh) [reply src/ArcGISRuntime.Samples.Shared/Managers/SampleManager.cs:100]: I've added an extra linq clause to order the featured samples when we create the SearchableTreeNode

`SearchableTreeNode featured = new SearchableTreeNode("Featured", AllSamples.Where(sample => featuredSamples.Contains(sample.FormalName, StringComparer.OrdinalIgnoreCase)).OrderBy(sample => sample.SampleName));`

File: src/Forms/Shared/Samples/Map/DisplayOverviewMap/readme.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ The data used in this sample is the [OpenStreetMap Tourist Attractions for North
 
 ## Additional information
 
- This sample uses the overview map toolkit component which can be accessed using the Esri.ArcGISRuntime.Toolkit.Xamarin.Forms nuget package. The [toolkit](https://github.com/Esri/arcgis-toolkit-dotnet) can also be cloned and set up locally. For information about setting up the toolkit, visit the [repository](https://github.com/Esri/arcgis-toolkit-dotnet/blob/main/README.md).
```
Comment (nCastle1) [inline src/Forms/Shared/Samples/Map/DisplayOverviewMap/readme.md:34]: Change is unrelated to this PR

File: src/Forms/Shared/Samples/Map/DisplayOverviewMap/readme.metadata.json [hunk 3]
```diff
@@ -19,11 +19,11 @@
     ],
     "nuget_packages": {
         "Esri.ArcGISRuntime": "100.13.0",
+        "Esri.ArcGISRuntime.Toolkit.Xamarin.Forms": "100.13.0",
         "Esri.ArcGISRuntime.UWP": "100.13.0",
         "Esri.ArcGISRuntime.Xamarin.Android": "100.13.0",
         "Esri.ArcGISRuntime.Xamarin.Forms": "100.13.0",
-        "Esri.ArcGISRuntime.Xamarin.iOS": "100.13.0",
-        "Esri.ArcGISRuntime.Toolkit.Xamarin.Forms": "100.13.0"
+        "Esri.ArcGISRuntime.Xamarin.iOS": "100.13.0"
```
Comment (nCastle1) [inline src/Forms/Shared/Samples/Map/DisplayOverviewMap/readme.metadata.json:26]: Another unrelated change - maybe the script needs to be updated in a separate PR?

File: src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Map/DisplayOverviewMap/readme.metadata.json [hunk 4]
```diff
@@ -19,8 +19,8 @@
     ],
     "nuget_packages": {
         "Esri.ArcGISRuntime": "100.13.0",
-        "Esri.ArcGISRuntime.WPF": "100.13.0",
-        "Esri.ArcGISRuntime.Toolkit": "100.13.0"
+        "Esri.ArcGISRuntime.Toolkit": "100.13.0",
+        "Esri.ArcGISRuntime.WPF": "100.13.0"
```
Comment (nCastle1) [inline src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Map/DisplayOverviewMap/readme.metadata.json:23]: Another unrelated change

**Review by nCastle1 — CHANGES_REQUESTED** [review]
There are a few sample file changes that shouldn't be included in this PR. Otherwise the change looks great.

**Review by nCastle1 — APPROVED** [review]
